src/redux/bookStore.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/bookSlice';
+import categoriesReducer from './categories/categoriesSlice';
+
+export default configureStore({

```

Comment:
[OPTIONAL] You can also add middleware like logger and thunk:
- Middleware allows you to intercept every action sent to the reducer so you can make changes to the action or cancel the action. Middleware helps you with logging, error reporting, making asynchronous requests, and a whole lot more.
- Logger makes it easy to debug an application. By logging actions and states, it is easy to understand coding errors, network errors, and other forms of bugs that might come up during production.
- Thunk lets you call action creators that return a function instead of an action object. That function receives the store's dispatch method, which is then used to dispatch regular synchronous actions inside the function's body once the asynchronous operations have been completed. This is very helpful when you use async/await when calling APIs.

Firstly, you can install the logger and thunk:
```
npm install redux-logger redux-thunk
```

Then, import the middleware, logger, and thunk.
```
import { applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
```

And finally, your store will now look like this:
```
const store = createStore(
  reducer,
  compose(applyMiddleware(thunk, logger)),
);
```